mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAttachmentModule.java

```diff
@@ -48,7 +50,14 @@ public CassandraAttachmentModule() {
                     .addColumn(CassandraAttachmentTable.TYPE, text())
                     .addColumn(CassandraAttachmentTable.SIZE, bigint())
                     .withOptions()
-                    .comment("Holds attachment for fast attachment retrieval")));
+                    .comment("Holds attachment for fast attachment retrieval")),
+            new CassandraTable(CassandraAttachmentMessageIdTable.TABLE_NAME,
+                SchemaBuilder.createTable(CassandraAttachmentMessageIdTable.TABLE_NAME)
+                    .ifNotExists()
+                    .addPartitionKey(CassandraAttachmentMessageIdTable.ATTACHMENT_ID, text())
+                    .addColumn(CassandraAttachmentMessageIdTable.MESSAGE_IDS, set(text()))
+                    .withOptions()
```

Comment:
Please : 
 - Levelled compaction (we will have many update to the same row => we need to have them as much as possible to the same SSTables, and the cost associated is low as it is metadata)
 - Enable Row cache (after all it's just tiny metadata)

Reply:
done